### PR TITLE
Mojaray2k 5313 mdt introduction page work set up

### PR DESCRIPTION
**Pull request by mojaray2k**
## Description
This PR is for the setup of Form 2346 using the advance configuration provided by the VA. 

## Testing done
As a developer you should be able to check out branch `mojaray2k-5313-mdt-introduction-page-work-set-up` and navigate to [Form 2346 Introduction Page](http://localhost:3001/mdt/2346/introduction) and step through the multi step form. 

## Screenshots
Cursor_and_Form-2346___Veterans_Affairs

## Acceptance criteria
- [ ] Be able to step through the multi step form all the way to the Confirmation Page. 

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/disability-benefits/2346/app-entry.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import '../../../platform/polyfills';
+import './sass/mdt-introduction.scss';
+
+import startApp from '../../../platform/startup';
```
Comment (Mottie) [inline src/applications/disability-benefits/2346/app-entry.jsx]: You can get rid of the extra `../../../` in the path to the `platform` folder. It's in a bunch of the files.

File: src/applications/disability-benefits/2346/manifest.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "appName": "Form-2346",
+  "entryFile": "./app-entry.jsx",
+  "entryName": "mdt-introduction",
+  "rootUrl": "/mdt/2346",
+  "production": false
+}
```
Comment (Mottie) [inline src/applications/disability-benefits/2346/manifest.json]: Make sure to also include this block ([reference](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/documentation/src/pages/getting-started/common-tasks/new-application.mdx)):

```js
  "template": {
    "vagovprod": false
  }
```

This prevents the page from appearing in production. I think it also keeps the page from showing up in the search results.

**Review by Mottie — APPROVED** [review]
I had a few more comments to add about the `config/form.js` but I realize that this is a mock up and you really just want to get this into staging.

**Review by maharielrosario — COMMENTED** [review]
This looks good and ready to merge into 3614!

File: src/applications/disability-benefits/2346/manifest.json (hunk 2 above)
Comment (mojaray2k) [reply src/applications/disability-benefits/2346/manifest.json]: Just added it. Thanks

File: src/applications/disability-benefits/2346/app-entry.jsx (hunk 1 above)
Comment (mojaray2k) [reply src/applications/disability-benefits/2346/app-entry.jsx]: @Mottie On both line 1 and 4 I should just put `./platform/` etc?

**Comment by mojaray2k** [comment]
> I had a few more comments to add about the `config/form.js` but I realize that this is a mock up and you really just want to get this into staging.

@Mottie Yeah that file is going to change drastically in other PR's

**Pull request merged**